keyboards/ydkb/ydpm40/info.json

```diff
@@ -0,0 +1,136 @@
+{
+    "manufacturer": "yang",
+    "keyboard_name": "ydkb/ydpm40",
+    "maintainer": "xofyarg",
+    "bootloader": "bootloadhid",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": false,
+        "nkro": false,
+    },
+    "matrix_pins": {
+        "cols": ["B0", "B1", "B2", "B3", "B4", "B5", "C5", "C4", "C3", "C2", "C1", "C0"],
+        "rows": ["D4", "D5", "D6", "D7"]
+    },
+    "processor": "atmega328p",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0240",
+        "vid": "0xFEED"
+    },
+    "build": {
+        "lto": true
+    },
+    "layouts": {
+        "LAYOUT_ydp40": {
```

Comment:
This should be something like `LAYOUT_ortho` rather than containing keyboard name. Same goes for the other layout name too.